### ISSUE TITLE
[Feature Request] Disable debug log

### ISSUE THREAD

**Issue by htdag** [yuzu-emu/yuzu#6607]
Hi,

I know this feature very useful.
But maybe user (like me) don't want yuzu write log to disk,
and send log to OutputDebugString.

So, an option allow user to complete disable log is needed.

**Comment by Zeroksas** [comment]
You can disable it by leaving global log filter window empty.

**Comment by htdag** [comment]
@Zeroksas Tested with empty global log filter.
The log still write to file `user\log\yuzu_log.txt`, and via OutputDebugString.

**Comment by german77** [comment]
you want to stop writing anything at all?

**Comment by htdag** [comment]
@german77 Yes, I want to stop all logs. (no file, no spam OutputDebugString).

**Comment by german77** [comment]
Set debug mode to *:Critical It will not stop from writting but the file would be minimal. Take into account that support is only given to full logs

**Comment by htdag** [comment]
That why I open this issue.

**Comment by german77** [comment]
The size is only a few KB on critical why need to disable completely

**Comment by htdag** [comment]
Some game have many error.
Sometimes I run yuzu with other programs that output using OutputDebugString too.
It's disturb, I can use filter but it's not convenient.
And because no way to disable.
